debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -882,6 +882,15 @@ public static EventProcessingFailureHandlingMode parse(String value) {
                     "The value of those properties is the select statement to use when retrieving data from the specific table during snapshotting. " +
                     "A possible use case for large append-only tables is setting a specific point where to start (resume) snapshotting, in case a previous snapshotting was interrupted.");
 
+    public static final Field SNAPSHOT_DELAY_MINUTES = Field.createInternal("snapshot.delay.minutes")
```

Comment:
Could you change it into "snapshot.delay.ms"? We handle other temporal options as milli-seconds, too, so it'd be good to be consistent.

Reply:
I was originally going to go with ms but thought that is too granular for the use case. But I can make this change to be consistent with other configurations.